ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -7,12 +7,12 @@
 import com.microsoft.azure.kusto.data.exceptions.OneApiError;
 import com.microsoft.azure.kusto.ingest.exceptions.IngestionClientException;
 import com.microsoft.azure.kusto.ingest.exceptions.IngestionServiceException;
+import com.microsoft.azure.kusto.ingest.exceptions.OneApiError;
```

Comment:
*Just to explain why it happened*
Along the way, I had made a change which impacted this file (at some point, I had the AadAuthenticationHelper additionally throw a DataClientException), but have since removed it. When I impact a file and see minor fixes along the way, I make them. So I made those changes, but then reverted my substantive change to the AadAuthenticationHelper exception list.

*What to do*
This is really minor, yet an (minor) improvement. I suggest it's worth including it in this PR even though it's no longer relevant. If you object I'll revert.

Reply:
Ok. Keep it.
but this is a difficult enough PR without distractions :)